README.md

```diff
@@ -1,21 +1,15 @@
-# Choosealicense.com
+**[ChooseALicense.com](https://choosealicense.com)** aims to provide **accurate**, **non-judgmental**, and **understandable** information about **open source licenses** in order to **help people make informed decisions** about the projects they start, maintain, contribute to, and use.
```

Comment:
Non-Comprehensive? Or "Popular" open source licenses? Yes, we have the "add a license" criteria elsewhere, but looking at the "mission statement" alone, we'd best serve it by cataloging every possible license. Is that the intention?